R/04-webprocess-obj.R

```diff
@@ -61,9 +60,24 @@ setMethod(f="initialize",signature="webprocess",
             algorithm = .Object@algorithm,
             version = .Object@version,
             email = .Object@email,
-            wait = .Object@wait, ...)
+            wait = .Object@wait, 
+            sleep.time = .Object@sleep.time, ...)
             {
-
+            
+            #things that use package globals:
+            if(!length(url))
+              url <- gconfig('wps.url')
+            
+            if(!length(sleep.time))
+              sleep.time <- gconfig('sleep.time')
+            
+            if(!length(wait))
+              wait <- gconfig('wait')
+            
+            if(!length(email))
+              email <- gconfig('email')
+            
+            
```

Comment:
could it make sense to move lines 81-94 (except 92) into the prototype? The `initialize` call never changes them, right?